src/objects/zcl_abapgit_object_intf.clas.locals_imp.abap

```diff
@@ -461,37 +461,94 @@ CLASS lcl_aff_type_mapping IMPLEMENTATION.
 
 ENDCLASS.
 
+
 CLASS lcl_aff_serialize_metadata DEFINITION.
   PUBLIC SECTION.
 
     CLASS-METHODS serialize
       IMPORTING is_intf          TYPE data
```

Comment:
can this IS_INTF get the correct type?

Reply:
I guess so. At some point in time, there was an underlying interface, that had TYPE data